db/app.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express')
+const app = express()
+
+const {getTopics} = require('../db/controllers/nc-news-controller')
+
+app.get('/api/topics', getTopics)
+
+app.all('/*', (req, res) => {
+        res.status(404).send({mess: 'not found'})
+    })
+app.use((err, req, res, next) => {
+    if (err === 500) {
+        console.log(err, '500 >>>>>>>>>>>>>>>>>>')
+        res.status(500).send({mess: 'Internal Server Error'})
+    }
+    next(err)
+  });
```

Comment:
This error handler will be our last line of defence if it doesn't hit any other errors. It does not need a next block as there is no next to pass to. Make sure you remove any console.logs once they are no longer needed.

Reply:
removed console log and next(err)